src/coreclr/vm/arm64/asmhelpers.S

```diff
@@ -343,6 +343,32 @@ NESTED_END OnHijackTripThread, _TEXT
 
 #endif // FEATURE_HIJACK
 
+// void* PacStripPtr(void *);
```

Comment:
Something to think about for future PR - should we always authenticate instead of stripping the return address?